### PR TITLE
Truncate FR dabase settings when a feeder is off

### PR DESCRIPTION
**Pull request by aleksandrychev**
Ticket: ENT-7698

### CODE REVIEW

**Comment by aleksandrychev** [comment]
@cf-bottom jenkins

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7421/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7421/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7421/

File: cfe_internal/enterprise/federation/federation.cf [hunk 1]
```diff
@@ -319,10 +319,9 @@ bundle agent clean_when_off
       "has_cftransport_fcontext" expression => returnszero("$(paths.semanage) fcontext -l | grep $(home)", "useshell");
 
   commands:
-      am_superhub::
```
Comment (nickanderson) [inline cfe_internal/enterprise/federation/federation.cf:322]: So, with this class guard removed both remote_hubs and federated_reporting_settings tables will be truncated on both feeders and the superhub. Is that what we want?

Comment (aleksandrychev) [reply cfe_internal/enterprise/federation/federation.cf:322]: yes, right, we need to truncate them as part of the cleanup task

Comment (aleksandrychev) [reply cfe_internal/enterprise/federation/federation.cf:322]: on both

Comment (nickanderson) [reply cfe_internal/enterprise/federation/federation.cf:322]: OK, it smells a bit funny to me to truncate tables every execution.

**Review by nickanderson — APPROVED** [review]

**Comment by aleksandrychev** [comment]
@cf-bottom jenkins

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7422/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7422/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7422/

**Comment by aleksandrychev** [comment]
@cf-bottom jenkins

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7424/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7424/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7424/

**Pull request merged**